modules/getting-started-web-console-creating-secret.adoc

```diff
@@ -1,43 +1,52 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-web-console.adoc
+// * tutorials/dev-app-web-console.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-web-console-creating-secret_{context}"]
-= Creating a secret
+= Providing access to the database by creating a secret
 
-The `Secret` object provides a mechanism to hold sensitive information such as passwords, {product-title} client configuration files, private source repository credentials, and so on.
-Secrets decouple sensitive content from the pods. You can mount secrets into containers using a volume plugin or the system can use secrets to perform actions on behalf of a pod.
-The following procedure adds the secret `nationalparks-mongodb-parameters` and mounts it to the `nationalparks` workload.
+The `nationalparks` application needs information, such as the database name, username, and passwords, on how to access the MongoDB database. However, because this information is sensitive, you should not store it directly in the pod.
```

Comment:
```suggestion
The `nationalparks` application needs information, such as the database name, username, and passwords, to access the MongoDB database. However, because this information is sensitive, you should not store it directly in the pod.
```